### ISSUE TITLE
Update for 5.3?

### ISSUE THREAD

**Issue by jakeols** [vinkas0/firebase-auth-laravel#3]
Any chance you'll be updating for 5.3? 

I know that the AuthorizesResources trait has been merged with AuthorizesRequests, and AuthenticatesAndRegistersUsers no longer exists in 5.3 so there's definitely some breaking changes. Let me know if you plan on updating, I may have time to update in the next couple weeks as well. 

**edit: changes were actually pretty simple, created PR #4**

**Comment by oberoivarun** [comment]
@jakeols The links to how to setup the package are not working anymore. Could you possibly give some guidance here to you branch and how to set it up?

**Comment by jakeols** [comment]
Hey @oberoivarun , I haven't used this project in forever so I'm not even sure if the changes I made work with the current laravel version. If I recall correctly, the only difference here is that a couple of the controller classes were replaced in 5.3, so pretty much just changing the class names worked. When I have a little more time I can create a new project and see if I can upgrade this repo. I believe the auth controllers here are structured a bit differently as well and should probably be updated.
